src/chapters/control-flow-and-collections/array.rst

```diff
@@ -0,0 +1,146 @@
+Array
+=====
+
+We've been introduced to arrays in Java in :ref:`a previous lesson <array>`, 
+so let’s spend a moment comparing them to ``ArrayLists``. As we noted at that beginning 
+of this section, we are going to use the Java ``ArrayList`` type to store
```

Comment:
"...in the previous section..." or "... on the previous page...", since this sentence appears at the top of the page.